converters/fromarray/instrumentation.ts

```diff
@@ -0,0 +1,90 @@
+let instrumentationConverters = [];
+instrumentationConverters[Instrumentation.JsError] = jsErrorFromArray;
+instrumentationConverters[Instrumentation.MissingFeature] = missingFeatureFromArray;
+instrumentationConverters[Instrumentation.XhrError] = xhrErrorFromArray;
+instrumentationConverters[Instrumentation.TotalByteLimitExceeded] = byteLimitExceededFromArray;
+instrumentationConverters[Instrumentation.Teardown] = noDataFromArray;
+instrumentationConverters[Instrumentation.PerformanceStateError] = noDataFromArray;
+instrumentationConverters[Instrumentation.ClarityAssertFailed] = assertFailedFromArray;
+instrumentationConverters[Instrumentation.ClarityDuplicated] = clarityDuplicatedFromArray;
+instrumentationConverters[Instrumentation.ClarityActivateError] = clarityActivateErrorFromArray;
+instrumentationConverters[Instrumentation.ShadowDomInconsistent] = inconsistentShadowDomFromArray;
+
+export default instrumentationConverters;
+
+function jsErrorFromArray(jsErrorData: any[]): IJsErrorEventData {
```

Comment:
>jsErrorFromArray [](start = 9, length = 16)

It would be better to have the to/from code in one place to make it easier to check that it matches on both sides I did *not* check that this matches as I assume tests will catch errors.